### PR TITLE
Add the C73327 test case for validationg the entries count.

### PR DESCRIPTION
**Pull request by helloarys**
@turtle-fly @fub2

### CODE REVIEW

**Comment by helloarys** [comment]
@PaynePei

File: case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[
+    {}
+]
```
Comment (fub2) [inline case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.json]: Compaining no newline. Does it matter?

Comment (helloarys) [inline case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.json]: I will check it with Forrest.

File: case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            for obj_node in obj_rack.get_node_list():
+                self.log('INFO', 'Check node {} of rack {} ...'.
+                         format(obj_node.get_name(), obj_rack.get_name()))
+                obj_node.get_bmc().ipmi.ipmitool_standard_cmd('sel clear')
+                time.sleep(3)
+                ret, rsp = obj_node.get_bmc().ipmi.ipmitool_standard_cmd('sel info')
+                self.log('INFO', 'ret: {}'.format(ret))
+                self.log('INFO', 'rsp: \n{}'.format(rsp))
+
+                if not re.search('Entries(\s)*:(\s)*0', rsp):
```
Comment (PaynePei) [inline case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.py]: the number of sel entry should be 1 after clear, right? @sharkconi @turtle-fly @fub2

Comment (turtle-fly) [inline case/infrasim/virtual_bmc/T73327_idic_SelInfoEntriesCountCheck.py]: Hi, Payne, considering there might come more SEL after clear. The SEL entry count should be >=1. I'm OK with this change.

**Comment by turtle-fly** [comment]
:+1:

**Comment by PaynePei** [comment]
👍

**Pull request merged**